doc/unwanted-modules.md

```diff
@@ -0,0 +1,68 @@
+# Disabling and undeploying unwanted FOLIO modules
+
+
+## Background
+
+FOLIO is big. [Really big](https://www.goodreads.com/quotes/14434-space-is-big-you-just-won-t-believe-how-vastly-hugely). If you're working on developing a module for it, you are likely running against a virtual machine managed by Vagrant -- but such VMs now need a great deal of memory. For example, `mod-copycat` uses enough FOLIO backend modules that it can't run against the `folio/core` Vagrant box, and needs `folio/testing`, which needs more than 10 Gb RAM if it is to avoid an orgy of paging that will render the host box all but unusable.
+
+So to reduce the in-memory size of FOLIO, we want to identify modules that we know we don't need and shut them down.
+
+This is a multi-stage process:
+
+<!-- md2toc -l 2 -s 1 unwanted-modules.md -->
+* [1. Identify the modules running on the tenant of interest](#1-identify-the-modules-running-on-the-tenant-of-interest)
+* [2. Choose the ones that are not wanted](#2-choose-the-ones-that-are-not-wanted)
+* [3. Check that disabling will not have catastrophic consequences](#3-check-that-disabling-will-not-have-catastrophic-consequences)
+* [4. Disable these modules (and those that depend on them)](#4-disable-these-modules-and-those-that-depend-on-them)
+* [5. Undeploy those modules including those that depend on them](#5-undeploy-those-modules-including-those-that-depend-on-them)
+
+In the development scenario, there is almost always only a single tenant in play, `diku`, so we will make that simplifying assumption in this document. To make the examples concrete, we will also assume that Okapi is running on localhost port 9130.
+
+
+## 1. Identify the modules running on the tenant of interest
+
+You can find these at http://localhost:9130/_/proxy/tenants/diku/modules
```

Comment:
Those are modules _enabled_ for the tenant (and will include things that are less interesting, like UI modules). Actual _running_ modules are at `http://localhost:9130/_/discovery/modules`

Reply:
Good point — now changed.